catalog/core/catalog-core-api/src/main/java/ddf/catalog/content/data/ContentItem.java

```diff
@@ -83,14 +93,6 @@
     InputStream getInputStream() throws IOException;
 
     /**
-     * Return the file associated with the content item.
-     *
-     * @return the content item's associated file
-     * @throws IOException if the content cannot be resolved to be an actual file
-     */
-    File getFile() throws IOException;
```

Comment:
I agree with @coyotesqrl - this leaks an implementation detail and also provides a means for a client to modify the underlying resource without using the framework.